app/assets/javascripts/stories.js

```diff
@@ -1,25 +1,28 @@
-document.addEventListener("input", (e) => {
-  const updateMarkdown = () => {
-    document.querySelectorAll("[data-has-preview]").forEach((element) => {
```

Comment:
this was doing a request for each element with preview instead of the current one that changed